apps/condo/domains/property/components/AddressSuggestionsSearchInput.tsx

```diff
@@ -73,11 +71,11 @@ export const AddressSuggestionsSearchInput: React.FC<AddressSearchInputProps> =
     )
 
     const renderOption = useCallback(
-        (dataItem: TSelectItem, searchValue) => {
+        (dataItem: TSelectedAddressSuggestion, searchValue) => {
             return (
                 <Select.Option
                     style={{ direction: 'rtl', textAlign: 'left', color: grey[6] }}
-                    key={dataItem.value}
+                    key={JSON.stringify(dataItem)}
```

Comment:
positions of object fields are not stable, better to generate key manually

Reply:
It does not matter here, because we build a new list on every new search input.